tests/test_train.py

```diff
@@ -1,14 +1,21 @@
+from pathlib import Path
+
 import pytest
 from pyannote.database import FileFinder, get_protocol
 from pytorch_lightning import Trainer
 
+from pyannote.audio.models.embedding.debug import SimpleEmbeddingModel
 from pyannote.audio.models.segmentation.debug import SimpleSegmentationModel
 from pyannote.audio.tasks import (
+    MultiLabelSegmentation,
     OverlappedSpeechDetection,
     SpeakerDiarization,
+    SupervisedRepresentationLearningWithArcFace,
     VoiceActivityDetection,
 )
 
+CACHE_FILE_PATH = "./cache/cache_file"
```

Comment:
Can we avoid hard coding this? 
Would it make sense to use a `@pytest.fixture` for this as well?